app.js

```diff
@@ -240,14 +240,13 @@ d.run(() => {
 							}
 							return callback(null, Object.assign({}, peer, { ip: address }));
 						});
-					}
+					})
 				);
 
-				return async.parallel(peerDomainLookupTasks, (err, results) => {
-					if (err) {
-						return cb(err, appConfig);
-					}
-					appConfig.peers.list = results;
+				return async.parallel(peerDomainLookupTasks, (_, results) => {
+					appConfig.peers.list = results
```

Comment:
We definitely don't want to filter out the array of seed peers!

Reply:
Cooopy